modules/test/playwright/pages/layout-admin-web/PagesAdminPage.ts

```diff
@@ -232,6 +237,12 @@ export class PagesAdminPage {
 		);
 	}
 
+	async deleteWidgetPage(name: string) {
```

Comment:
If using `isolatedSite` works (look for comment below), we can remove this function.

Reply:
Thank you for checking it, I will review them tomorrow.